osl/interface_linux.go

```diff
@@ -307,14 +307,26 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 
 func configureInterface(iface netlink.Link, i *nwIface) error {
 	ifaceName := iface.Attrs().Name
+	address := func(i *nwIface) string {
+		if i.Address() == nil {
+			return "(no address)"
+		}
+		return fmt.Sprintf("%q", i.Address())
+	}
+	address6 := func(i *nwIface) string {
+		if i.AddressIPv6() == nil {
+			return "(no address)"
+		}
+		return fmt.Sprintf("%q", i.AddressIPv6())
+	}
 	ifaceConfigurators := []struct {
 		Fn         func(netlink.Link, *nwIface) error
 		ErrMessage string
 	}{
 		{setInterfaceName, fmt.Sprintf("error renaming interface %q to %q", ifaceName, i.DstName())},
 		{setInterfaceMAC, fmt.Sprintf("error setting interface %q MAC to %q", ifaceName, i.MacAddress())},
-		{setInterfaceIP, fmt.Sprintf("error setting interface %q IP to %q", ifaceName, i.Address())},
```

Comment:
> the runtime catches it (even in the original version),

Yes I was wondering about that. Because `setInterfaceIPv6()` is called pretty frequently with a nil address.

I am fine to change it to `%v`.